airgun/entities/subscription.py

```diff
@@ -103,6 +103,24 @@ def read_delete_manifest_message(self):
             manage_view.close_button.click()
         return delete_message
 
+    def read_expire_manifest_message(self):
+        """Read message displayed about 'manifest expiration' at Subscription Manifest section"""
+        expire_manifest_header = None
+        expire_manifest_message = None
+        view = self.navigate_to(self, 'Manage Manifest')
+        view.wait_animation_end()
+        if view.manifest_alert.is_displayed:
+            expire_manifest_header = view.manifest_expire_header.read()
+            expire_manifest_message = view.manifest_expire_message.read()
+        else:
+            # Subscription Manifest header & message is not present
+            expire_manifest_message = 'Manifest expire alert not found'
```

Comment:
Should you raise as exception here ?

Reply:
I think it's not needed, becuase inside test case we are updating settings for `expire soon days` and re-calling same function to get expected message (that time if block would executes)